src/components/views/messages/BridgeError.js

```diff
@@ -0,0 +1,139 @@
+        room: PropTypes.object.isRequired,
+    };
+
+    constructor(props) {
+        super(props);
+        // this.state = {}
+    }
+
+    /**
+     * Returns all bridge error relations for this event.
+     *
+     * @returns {MatrixEvent[]}
+     */
+    _getRelations() {
+        const { mxEvent, room } = this.props;
+
+        // const room = matrixClient.getRoom(mxEvent.getRoomId());
+        const timelineSet = room.getUnfilteredTimelineSet();
+        const relations = timelineSet.getRelationsForEvent(
+            mxEvent.getId(),
+            "m.reference",
+            "de.nasnotfound.bridge_error",
+        );
+
+        return relations ? relations.getRelations() : [];
+    }
+
+    /**
+     * Returns a list of all users matched by the regex at the time of the event.
+     *
+     * @param {string} regexStr
+     * @returns {RoomMember[]}
+     */
+    _findMembersFromRegex(regexStr) {
+        const { room } = this.props;
+        const regex = new RegExp(regexStr);
+
+        // TODO[V02460@gmail.com]: Get room state at the proper point in time
+        const roomState = room.getLiveTimeline().getState(EventTimeline.FORWARDS);
+        const members = roomState.getMembers();
+
+        return members.filter(m => regex.test(m.userId));
+    }
+
+    /**
+     * Returns the network name and the affected users for the given relation.
+     *
+     * @param {MatrixEvent} relation
+     * @returns {{networkName: string, affectedUsers: RoomMember[]}}
+     */
+    _getRelationInfo(relation) {
+        if (!relation.event || !relation.event.content) {
+            return { networkName: "", affectedUsers: [] };
+        }
+        const content = relation.event.content;
+
+        const networkName = (typeof content.network_name === 'string' ?
+            content.network_name :
+            ""
+        );
+        const affectedUsersRegex = (Array.isArray(content.affected_users) ?
+            content.affected_users :
+            []
+        );
+        const affectedUsers = affectedUsersRegex.flatMap(
+            this._findMembersFromRegex.bind(this),
+        );
+
+        return { networkName, affectedUsers };
+    }
+
+    /**
+     * Returns the rendered string for the given relation.
+     *
+     * @param {{networkName: string, affectedUsers: RoomMember[]}} relationInfo
+     * @return {string}
+     */
+    _renderInfo(relationInfo) {
+        const usernames = relationInfo.affectedUsers.map(u => u.name).join(", ");
+
+        if (relationInfo.networkName) {
+            return _t(
+                "%(networkName)s: %(affectedUsers)s",
+                {
+                    networkName: relationInfo.networkName,
+                    affectedUsers: usernames,
+                },
+            );
+        } else {
+            return _t(
+                "Affected users: %(affectedUsers)s",
+                { affectedUsers: usernames },
+            );
+        }
+    }
+
+    render() {
```

Comment:
View updates are not working properly yet.

- The view refresh of the failed message must be linked to the arrival of the bridge error event.
- The list of affected users is incomplete before any foreign refresh takes place.